app/(header-default)/documents/[slug]/_components/conformite-section.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+import Conformite from '#components/espace-agent-components/conformite';
+import { AsyncDataSectionClient } from '#components/section/data-section/client';
+import { TwoColumnTable } from '#components/table/simple';
+import { EAdministration } from '#models/administrations/EAdministration';
+import { IUniteLegale } from '#models/core/types';
+import useFetchConformite from 'hooks/fetch/conformite';
+
+function ConformiteSection({ uniteLegale }: { uniteLegale: IUniteLegale }) {
+  const conformite = useFetchConformite(uniteLegale);
+
+  return (
+    <AsyncDataSectionClient
```

Comment:
No, I used the former because it's clearer, but `DataSectionClient` is still exported for compatibility. Once we get rif of `AsyncDataSectionServer`,  we'll decide on which to keep.